docs/modules/ic-identity-guide/pages/auth-how-to.adoc

```diff
@@ -6,21 +6,23 @@
 
-NOTE: On macOS/iOS/ipadOS (including iPhones), authentication using Safari is coupled to your browser profile. If you want to log in to an application in a different browser, or if you use multiple Safari browser profiles, you have to add the combination of your authentication method and the new browser as a new device. See: <<Add a device,`+Add a device+`>>.
+When you access an application that uses Internet Identity, you will be re-directed to the Internet Identity app. Once you authenticate into the app, Internet Identity creates a unique identity that is saved in the application's persistance layer (or web server), but does not contain any personal information, because at no point have you offered any personal information. However, this unique and anonymous identity is associated with your public ID number, so if you lose the number, you will also lose the account identity associated with it.
 
-Windows users can only use a security key as an authorized device.
+NOTE: On OSX (including iPhones), authentication using Safari is coupled to your browser profile. If you want to log in to an application in a different browser, or if you use multiple Safari browser profiles, you have to add the combination of your authentication method and the new browser as a new device. See: <<Add a device,`+Add a device+`>>.
+
+Windows Hello users can only use a security key as an authorized device.
 
 Firefox does not currently accept OSX with any device authentication method other than a security key. 
 
 == Create an Internet Identity
 
-You can securely access applications running on the {IC} that use Internet Identity for authentication, by selecting an authentication method and having an Internet Identity ID number generated for you. You can create as many identities as you want by registering new ID numbers. 
+You can securely access applications running on the {IC} that use the Internet Identity for authentication, by selecting an authentication method and having an Internet Identity ID number (user number) generated for you. You can create as many identities as you want by registering new ID numbers. 
 
-When you first access an application running on the {IC}, you are directed to Internet Identity and asked to enter a ID number to log in. If you are a new user, you need to register to obtain an ID number.
+When you first access an application running on the {IC}, you are directed to Internet Identity and asked to enter a User Number to log in. If you are a new user, you need to register to obtain a user number.
```

Comment:
User number is deprecated and is now named "ID number".

Reply:
Again, happy to change this when the ICIdentity app incorporates this change.